Gemfile.lock

```diff
@@ -157,7 +157,7 @@ DEPENDENCIES
   webpacker
 
 RUBY VERSION
-   ruby 2.5.1p57
+   ruby 2.3.1p112
```

Comment:
?

Reply:
My bad, my local docker image was old, it's getting rebuilt